src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
I should add two more methods for searching/getting annotation sets. I will send a PR today. I can also take a stab at wiggle methods but this will take more time and I really think it would be easier to propose them in a follow-up PR as they are independent of the other genomic features.

On `SearchAnnotations`, there is one primary unresolved issue which is around searching 'type'. The `OntologyTerm` as a search parameter doesn't really make sense; you'd need to define some very specific rules about how matching works and how subfields should be populated. I guess I would prefer that there were just and ontology ID (as there is a reference ID) or else an enum of types which would yield a more natural query. This might be a rathole though, so the simplest solution is to remove the search by type functionality from the initial PR.

Beyond that, my only other major concern is around naming, which uses an abbreviated style which is inconsistent with the rest of the GA4GH APIs. Would probably favor just calling them `features` or something less wordy. This might be a better issue to discuss on the pull request. I have a few other gripes with the current branch, but it will be more effective for me to state them as comments on the main GA4GH PR; github issues are a very bad way of doing a code review.